modules/openapi-generator/src/main/resources/typescript-fetch/apis.mustache

```diff
@@ -110,8 +110,17 @@ export class {{classname}} extends runtime.BaseAPI {
         if (this.configuration && (this.configuration.username !== undefined || this.configuration.password !== undefined)) {
             headerParameters["Authorization"] = "Basic " + btoa(this.configuration.username + ":" + this.configuration.password);
         }
-
+        
         {{/isBasic}}
+        {{#isBasicBearer}}
+        if (this.configuration && (this.configuration.accessToken || this.configuration.apiKey)) {
+            const token = this.configuration.accessToken || this.configuration.apiKey;
```

Comment:
```suggestion
            const token = this.configuration.accessToken || this.configuration.apiKey;
            const tokenString = typeof token === 'function' ? token("{{name}}", [{{#scopes}}"{{{scope}}}"{{^-last}}, {{/-last}}{{/scopes}}]) : token;
```
since 
https://github.com/OpenAPITools/openapi-generator/blob/eb8c38ca1f6b7f8a08d5bd956eec5c38cac63d11/modules/openapi-generator/src/main/resources/typescript-fetch/runtime.mustache#L122
see also 
https://github.com/OpenAPITools/openapi-generator/blob/eb8c38ca1f6b7f8a08d5bd956eec5c38cac63d11/modules/openapi-generator/src/main/resources/typescript-rxjs/apis.mustache#L128-L135